linux_os/guide/system/software/integrity/software-integrity/aide/aide_use_fips_hashes/oval/shared.xml

```diff
@@ -27,7 +27,7 @@
   version="1">
     <ind:filepath>/etc/aide.conf</ind:filepath>
     <ind:pattern operation="pattern match">^[A-Z]*[\s]*=[\s]*.*(sha1|rmd160|sha256|whirlpool|tiger|haval|gost|crc32).*$</ind:pattern>
-    <ind:instance datatype="int" operation="greater than or equal">0</ind:instance>
+    <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
```

Comment:
Why this change?

Reply:
This should be fine, the instance count starts at `1`.
https://github.com/OVALProject/Language/blob/master/docs/independent-definitions-schema.md#-textfilecontent54_object-

Reply:
This has been already implemented in another PR.